docs/how-to/multicluster-loadbalanced-dnspolicy.md

```diff
@@ -111,7 +111,8 @@ The health check section is optional, the following fields are available:
 - `port`: The port to connect to
 - `protocol`: The protocol to use for this connection
 
-For more information about DNS Health Checks, see [this guide](../dnspolicy/dns-health-checks.md).
+[//]: # (ToDo mnairn)
```

Comment:
is this so we don't break docs when me merge?

Reply:
meh, i need to go back through the docs and link to to where they are now

Reply:
Updated